docs/storage_size_calculator.md

```diff
@@ -0,0 +1,102 @@
+# Install Storage Size Calculator on OpenShift
+
+This document demonstrates how to install storage size calculator for loki on OpenShift.
+
+_Note:_ The storage size calculator works on **OpenShift** only!
```

Comment:
I wouldn't say it works only! I recommend you say `works out of the box on OpenShift`. For other distribution we should list the availability of required services, e.g. prometheus, scrape configuration for log-file-metric exporter, promsecret to access the custom prometheus URL.